components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -523,7 +526,13 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 		}
 
 		if (e.detail.isSource && this.__getParentViewFromEvent(e) === this) {
-			e.detail.sourceView.__dispatchViewResize();
+			if (useResizeFix) {
```

Comment:
Although trivial, this is being flagged due to uncertainty of side-effects associated with delaying the event by a frame in other contexts.